content/influxdb/v2.0/reference/flux/stdlib/dict/remove.md

```diff
@@ -0,0 +1,59 @@
+---
+title: dict.remove() function
+description: >
+  The `dict.remove()` function removes a key value pair from a dictionary and returns
+  and updated dictionary.
+menu:
+  influxdb_2_0_ref:
+    name: dict.remove
+    parent: Dictionary
+weight: 301
+---
+
+The `dict.remove()` function removes a key value pair from a dictionary and returns
+and updated dictionary.
+
+```js
+import "dict"
+
+dict.remove(
+  dict: [1: "foo", 2: "bar"],
+  key: 1
+)
+```
+
+## Parameters
+
+<p>
+  {{< req "All paremeters are required" >}}
+</p>
+
+### dict
+Dictionary to remove a key-value pair from.
+
+_**Data type:** Dictionary_
+
+### key
+Key to remove from the dictionary.
+Must be the same type as existing keys in the dictionary.
+
+_**Data type:** String | Boolean | Integer | Uinteger | Float | Time | Bytes_
+
+## Examples
+
```

Comment:
Shall we add any comments? 
// Remove key 1
// Verify key 1 is empty
// Verify key 2 still has value 

Also, would this be single example? Notice we have varying patterns across.